db/connection.go

```diff
@@ -0,0 +1,43 @@
+package db
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/jinzhu/gorm"
+	_ "github.com/lib/pq"
```

Comment:
This can't be here if 'postgres' is an optional thing. Main or Test control that.